src/templates/manager/preview_form.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% load static from staticfiles %}
+{% load bootstrap3 %}
+
+{% block body %}
+<div class="row well well-sm">
+<form enctype="multipart/form-data" method="post">
+  <div class="col-md-12">
+    <h3>Preview of {{ form.name }}</h3>
+    {% if form_type == 'proposal' %}
+      {% bootstrap_form default_fields %}
+    {% endif %}
+  </div>
+  {% for field in preview_form %}
+      <div class="{% if field.field.widget.attrs.div_class %}
+                    {{ field.field.widget.attrs.div_class }}
+                  {% else %}
+                    col-md-12
+                  {% endif %}">
+          {% if field.field.widget.attrs.is_checkbox %}
+            {% if field.errors %}
+                <span class="red">{{ field.errors|join:", " }}</span>
+                <br />
+            {% endif %}
+            {{ field }} &nbsp;
+              {% if field.field.help_text %}
+                  {{ field.field.help_text|safe }}
+              {% endif %}
+          {% else %}
+             {% if field.errors %}
+                 <span class="red">{{ field.errors|join:", " }}</span>
+             {% endif %}
+             {% bootstrap_field field %}
+          {% endif %}
+      </div>
+  {% endfor %}
+</form>
```

Comment:
```</form>``` end tag should be one more indentation in from the end tag ```</div>``` below it. I suspect you're problems arrise from the previous developers who didn't believe in formatting template code